assigment_1_Q1.py

```diff
@@ -0,0 +1,3 @@
+def check_permutation(str1, str2):
+    return sorted(str1.lower())==sorted(str2.lower())
```

Comment:
This file is missing all sorts of testing. Please include a function that performs tests.